grpc/sa-wrappers.go

```diff
@@ -485,7 +485,9 @@ func (sas StorageAuthorityServerWrapper) GetValidAuthorizations(ctx context.Cont
 		if err != nil {
 			return nil, err
 		}
-		resp.Valid = append(resp.Valid, &sapb.ValidAuthorizations_MapElement{Domain: &k, Authz: authzPB})
+		// Make a copy of k because it will be reasigned with each loop.
```

Comment:
Additionally, @jsoref points out that you've typo'd "reasigned" both places. It should be "reassigned". Can you fix that within this PR? I'm closing https://github.com/letsencrypt/boulder/pull/2501 to keep things simple since this isn't merged yet.